python/stempy/io/__init__.py

```diff
@@ -162,6 +162,31 @@ def save_electron_counts(path, events, scan_dimensions, frame_dimensions=None):
 
         frames[...] = events
 
+def read_electron_counts(path):
```

Comment:
I think this should be `load_electron_counts` ( as in save/load )

Reply:
Done